File: app/src/main/java/com/bigbang/googlebooksfirebase/util/DebugLogger.java
package com.bigbang.googlebooksfirebase.util;

import android.util.Log;
import static com.bigbang.googlebooksfirebase.util.Constants.*;


public class DebugLogger {

    public static void logError(Throwable throwable) {
        Log.d(TAG, ERROR_PREFIX + throwable.getLocalizedMessage());
    }

    public static void logDebug(String message) {
        Log.d(TAG, message);
    }
}

File: app/src/main/java/com/bigbang/googlebooksfirebase/util/DebugLogger.kt
package com.bigbang.googlebooksfirebase.util

import android.util.Log
import com.bigbang.googlebooksfirebase.util.Constants.ERROR_PREFIX
import com.bigbang.googlebooksfirebase.util.Constants.TAG

object DebugLogger {
    @JvmStatic
    fun logError(throwable: Throwable) {
        Log.d(TAG, ERROR_PREFIX + throwable.localizedMessage)
    }

    @JvmStatic
    fun logDebug(message: String?) {
        Log.d(TAG, message)
    }
}